python/ray/dataframe/dataframe.py

```diff
@@ -3,6 +3,14 @@
 from __future__ import print_function
 
 import pandas as pd
+from pandas.api.types import is_scalar
+from pandas.util._validators import validate_bool_kwarg
+from pandas.core.index import _ensure_index_from_sequences
+from pandas._libs import lib
+from pandas.core.dtypes.cast import maybe_upcast_putmask
+from pandas.compat import lzip
+
+import warnings
```

Comment:
Should we be using the warnings module everywhere else in the codebase? Right now we're just printing warnings to stdout, but that's almost certainly the wrong approach.

Reply:
Probably. We are using `warnings` here because that's what Pandas does. Warnings are typically sent to stderr. They also show up differently in a Jupyter Notebook.